rest/rest_collection.cc

```diff
@@ -752,6 +768,7 @@ getState() const
     case State::_executing:
         return L"executing";
     default:
+        ExcAssert(!"update the BackgroundTaskBase::getState");
```

Comment:
Only you know the meaning of that message, I don't understand it :) .  Is it a logic error?  Unimplemented code?  If so, what does it need to do?  etc.